docs/modules/ROOT/pages/exporting.adoc

```diff
@@ -45,3 +47,110 @@ You should only need to change `hello.java` to match your application/script.
 
 You can read more about how jitpack handle builds at https://jitpack.io/docs/BUILDING/.
 
+== Exporting as a project
+
+If you want to transition your developed script to a full-scale Java project, you can use `jbang export gradle|maven <script>` to export it as a Gradle or Maven project. If you have specified multiple sources or resources using `//SOURCES` or `//FILES` in the script, they will also be exported. Tags such as `//JAVA`, `//DEPS`, `//REPOS`, `//GAV`, and `//DESCRIPTION` will also be reflected in the exported `build.gradle` or `pom.xml`.
+
+Unless explicitly specified via command options or tags, the GAV of the exported project defaults to `org.example:<script name>:999-SNAPSHOT`.
+
+Let's say you want to export the following `hello.java` to a project:
+
+[source,java]
+----
+///usr/bin/env jbang "$0" "$@" ; exit $?
+
+//SOURCES model/Message.java
+//FILES application.properties
+//DEPS org.slf4j:slf4j-simple:2.0.17
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import model.Message;
+
+public class hello {
+    private static final Logger log = LoggerFactory.getLogger(hello.class);
+
+    public static void main(String... args) {
+        log.info("{}", new Message());
+    }
+}
+----
+
+=== To a Gradle project
+
+You can export the script to a Gradle project with the GAV `dev.jbang:myapp:1.0.0-SNAPSHOT` as follows:
+
+[source,bash]
+----
+jbang export gradle --group dev.jbang --artifact myapp --version 1.0.0-SNAPSHOT hello.java
```

Comment:
```suggestion
jbang export gradle --group org.acme --artifact myapp --version 1.0.0-SNAPSHOT hello.java
```

lets not confuse use of the jbang namespace.

Reply:
Done